_posts/2024-11-14-OpenSearchCon-2024-NA-Blog.md

```diff
@@ -0,0 +1,51 @@
+* Session: ![Lucene And Beyond - Core Storage Extension In OpenSearch](https://youtu.be/-_mXJqvPw0o?si=7Lb1Y__uAZ7ZCc66)
+    OpenSearch is tightly bound to the Lucene core APIs that facilitate Encoding, Transactions, Merges, Search. This session, led by Samuel Herman, focused on how the OpenSearch storage encoding can be extended to popular formats (e.g. Parquet, Avro) that are readable by public big data systems such as Apache spark. 
+* Session: ![Bringing cloud native innovation from search platforms at Uber and Slack into OpenSearch](https://youtu.be/iZrFVqvTD9s?si=loButiXx8zm2PHob)
+    This session, led by Karthikeyan Ramasamy and Bryan Burkholder, showcased how their organizations’ unique requirements of low latency, high throughput workloads led them to develop their own search and log analytics platforms, making it easy to operate, cost-effective, and scale to petabytes of data. The session covers how Uber and Slack are collaborating with OpenSearch Project to bring some of their learnings and innovations from their platforms into OpenSearch, with special focus on a modular cloud native architecture with isolation between readers and writers and a performant messaging protocol for communication between services. 
+* Session: ![A deep dive into OpenSearch Serverless](https://www.youtube.com/watch?v=JPo_uF4Fw4M)
+    This session led by Rohit Nair and Milav Shah, dives deep into how OpenSearch serverless works, including architecture details, understanding performance and operational characteristics and future roadmap.
+
+4. Closing Remarks: What’s Next for OpenSearch?
+
+The event was not just about technical sessions—it was also a fantastic opportunity to network with peers in the open-source search space. The community lounge was buzzing with activity, and there were plenty of informal meetups where attendees could connect with OpenSearch engineers, contributors, and advocates.
+
+As the OpenSearchCon event drew to a close, it was clear that the conference was a significant milestone for the OpenSearch community. The announcements around new features, improvements in observability, and the ongoing commitment to open-source principles left attendees excited about the future of the project.
+
+The OpenSearch roadmap for 2024 highlights key themes around which the contributors are innovating, including stability, availability, and resiliency (StAR); cost, performance, and scale (CoPS); ease of use; observability and log analytics; search and machine learning; security; and a modular architecture.
+
```

Comment:
Line 51: Should we provide a link to the events page that we invite the reader to bookmark?

Reply:
@prthsh2020 I think so, but maybe have @pajuric confirm.

Reply:
Change made